openedx_webhooks/webhook_confs.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Configuration for webhooks we want to install.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from flask import url_for
+
+
+WEBHOOK_CONFS = ({
+    'config': {
+        'url': url_for("github_views.pull_request", _external=True),
+        'content_type': 'json',
+        'insecure_ssl': False,
+    },
+    'events': ['pull_request']
+})
+"""
+Tuple[Dict[
```

Comment:
The code above isn't a tuple of dicts. It's a parenthesized dict. (Lists are better for these anyway :)  )